man/fit.cosh.Rd

```diff
@@ -38,13 +38,6 @@ plot.coshfit(x, col.fitline="black", plot.mass=TRUE, plot.corr=FALSE, ...)
   \item{useCov}{
     Use the correlated chisquare. This works only for not too noisy data.
   }
-  \item{replace.na}{
```

Comment:
is there are reason for the disappearance here?

Reply:
The function does not have that argument, so I have updated the documentation accordingly.

So it might be the question whether the function should have that argument. I am against that, we should have a separate function that fills the NA values in an effective mass.

Reply:
I agree.